src/Views/page.js

```diff
@@ -100,6 +101,21 @@ class PageView extends BaseView {
 			{icon:"✏️", label:"edit", href:"#editpage/"+page.id},
 			{icon:"🗂️", label:"childs", href:"#category/"+page.id},
 		])
+
+		if (!OPTS.has('dev')) {
+			const domain = Req.server.split("/")[0]
+			let href = undefined
+			if (Object.hasOwn(page.values, "share") && page.values.share) {
```

Comment:
why are we checking page.values.share twice? (i would just use if (page.values.share))

Reply:
it's to check if it's also a truthy value

Reply:
ok yes i can do that, pushing a change